commons-extension-webapp/src/main/webapp/ckeditorCustom/contents.less

```diff
@@ -0,0 +1,291 @@
+/*
+Copyright (c) 2003-2015, CKSource - Frederico Knabben. All rights reserved.
+For licensing, see LICENSE.md or http://ckeditor.com/license
+*/
+
+@import (css) "/rest/v1/platform/branding/css";
+@import "../ckeditor/skins/common/editor.less";
+
+@font-face {
+  font-family: 'PLF-FONT-ICONS';
+  src: url('/platform-ui/skin/fonts/PLF-FONT-ICONS.eot?-m9uidt');
+  src: url('/platform-ui/skin/fonts/PLF-FONT-ICONS.eot?#iefix-m9uidt') format('embedded-opentype'), url('/platform-ui/skin/fonts/PLF-FONT-ICONS.ttf?-m9uidt') format('truetype'), url('/platform-ui/skin/fonts/PLF-FONT-ICONS.woff?-m9uidt') format('woff'), url('/platform-ui/skin/fonts/PLF-FONT-ICONS.svg?-m9uidt#PLF-FONT-ICONS') format('svg');
+  font-weight: normal;
+  font-style: normal;
+}
+
+body {
+  /* Font */
+  font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
```

Comment:
No needed since the CKEditor should reuse the default `font-family` defined in portal style.

```suggestion
```